docs/README.md

```diff
@@ -1,35 +1,44 @@
 # TorchServe
 
-TorchServe is a flexible and easy to use tool for serving PyTorch models.
+TorchServe is a performant, flexible and easy to use tool for serving PyTorch eager mode and torschripted models.
 
 ## Basic Features
 
 * [Serving Quick Start](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model) - Basic server usage tutorial
 * [Model Archive Quick Start](https://github.com/pytorch/serve/tree/master/model-archiver#creating-a-model-archive) - Tutorial that shows you how to package a model archive file.
 * [Installation](https://github.com/pytorch/serve/blob/master/README.md#install-torchserve) - Installation procedures
-* [Serving Models](server.md) - Explains how to use torchserve
-   * [REST API](rest_api.md) - Specification on the API endpoint for TorchServe
+* [Serving Models](server.md) - Explains how to use TorchServe
+* [REST API](rest_api.md) - Specification on the API endpoint for TorchServe
 * [Packaging Model Archive](https://github.com/pytorch/serve/tree/master/model-archiver#torch-model-archiver-for-torchserve) - Explains how to package model archive file, use `model-archiver`.
 * [Inference API](inference_api.md) - How to check for the health of a deployed model and get inferences
 * [Management API](management_api.md) - How to manage and scale models
 * [Logging](logging.md) - How to configure logging
 * [Metrics](metrics.md) - How to configure metrics
-   * [Metrics API](metrics_api.md) - How to configure metrics API
+* [Prometheus and Grafana metrics](metrics_api.md) - How to configure metrics API with Prometheus formatted metrics in a Grafana dashboard
+* [Captum Explanations](https://github.com/pytorch/serve/blob/master/captum/Captum_visualization_for_bert.ipynb) - Built in support for Captum explanations for both text and images
 * [Batch inference with TorchServe](batch_inference_with_ts.md) - How to create and serve a model with batch inference in TorchServe
-* [Model Zoo](model_zoo.md) - List of pre-trained model archives ready to be served for inference with TorchServe.
-* [Examples](https://github.com/pytorch/serve/tree/master/examples) - Many examples of how to package and deploy models and workflows with TorchServe
-
-## Advanced Features
-
-* [Advanced configuration](configuration.md) - Describes advanced TorchServe configurations.
-* [Custom Service](custom_service.md) - Describes how to develop custom inference services.
-* [Unit Tests](https://github.com/pytorch/serve/tree/master/ts/tests#testing-torchserve) - Housekeeping unit tests for TorchServe.
-* [Benchmark](https://github.com/pytorch/serve/tree/master/benchmarks#torchserve-model-server-benchmarking) - Use JMeter to run TorchServe through the paces and collect benchmark data.
-* [TorchServe on Kubernetes](https://github.com/pytorch/serve/blob/master/kubernetes/README.md#torchserve-on-kubernetes) -  Demonstrates a Torchserve deployment in Kubernetes using Helm Chart.
 
-## Default Handlers
+## Examples
```

Comment:
Need to keep "Default Handlers" section for the built in handlers. There can be a separate section for Examples at the end